### ISSUE TITLE
Allow specifying region for AWS workers

### ISSUE THREAD

**Issue by yaroslavvb** [studioml/studio#240]
Region seems to be hardwired right now

https://github.com/studioml/studio/blob/28fe412a629b24d327c25e9c7a787b758417a82a/studio/ec2cloud_worker.py#L73

 self.region = 'us-east-1'

**Comment by pzhokhov** [comment]
yeah, we'll need to read it from boto client. I'll fix this as we set up a CI build that tests cloud workers as well (turns out it is not so simple - public travis builds expose environment variables with credentials easily, so we need a private build for that)

**Comment by yaroslavvb** [comment]
private build is also nicer because Travis doesn't support 14.04, or GPUs. At brain we used jenkins

**Comment by pzhokhov** [comment]
yeah :) the main downside is I need to go through the company access rights and bureaucracy - with public build it is way easier

**Comment by pzhokhov** [comment]
Turns out that AMIs are also regional. Need to provide region-dependent ami selection, pricing etc.
